treeherder/config/settings.py

```diff
@@ -444,6 +444,8 @@
     'linux': 200,
 }
 RESET_BACKFILL_LIMITS = timedelta(hours=24)
+PERF_SHERIFF_BOT_CLIENT_ID = env('PERF_SHERIFF_BOT_CLIENT_ID', default=None)
```

Comment:
A comment line would do well here. Something like "Taskcluster credentials for PerfSheriffBot"?

Reply:
OK